lib/src/base-styles/_nx-variables.scss

```diff
@@ -51,6 +51,11 @@
   // similar to above, but for use in drop-shadow filters rather than box-shadow properties
   --nx-drop-shadow-focus: 0 0 3px var(--nx-color-interactive-shadow-focus);
 
+  // for dropdown 
+  --nx-box-shadow-dropdown: 0 2px 12px 0 rgba(74, 77, 113, 0.2),
```

Comment:
Please respect the existing ordering here too by moving this up to be with the other `box-shadow` and fixing the comment on `--nx-drop-shadow-focus` to reflect its new place

Reply:
Updated here c88f1abdb10667054bc42ea3d8ef210e6d81eaf1.